lib/vrl/compiler/src/expression/op.rs

```diff
@@ -191,7 +191,7 @@ impl Expression for Op {
                 let td = TypeDef::float();
 
                 // Division is infallible if the rhs is a literal normal float or integer.
-                match self.rhs.as_value() {
+                match dbg!(self.rhs.as_value()) {
```

Comment:
woops

Reply:
Hrm, I'm wondering why clippy didn't catch this with `-Dclippy::dbg_macro` in `.cargo/config`.

Reply:
Figured out the issue: https://github.com/vectordotdev/vector/pull/11674